### PR TITLE
PMM-7338 Add API for changing security check interval

### PR DESCRIPTION
**Pull request by palash25**

### CODE REVIEW

File: api/managementpb/checks.proto [hunk 1]
```diff
@@ -37,6 +47,13 @@ message ChangeSecurityCheckParams {
   bool disable = 3;
 }
 
+// ChangeSecurityCheckIntervalParams specifies interval change parameters for a single check.
+message ChangeSecurityCheckIntervalParams {
+  // The name of the check to change interval of.
+  string name = 1;
+  SecurityCheckInterval interval = 2;
```
Comment (artemgavrilov) [inline api/managementpb/checks.proto:54]: Missing comment

File: api/managementpb/checks.proto [hunk 2]
```diff
@@ -59,6 +76,12 @@ message ChangeSecurityChecksRequest {
 
 message ChangeSecurityChecksResponse {}
 
+message ChangeSecurityChecksIntervalRequest {
+  ChangeSecurityCheckIntervalParams param = 1;
```
Comment (artemgavrilov) [inline api/managementpb/checks.proto:80]: Why do we need this `ChangeSecurityCheckIntervalParams`? Why not just add required fields to the `ChangeSecurityChecksIntervalRequest` directly?

Comment (palash25) [reply api/managementpb/checks.proto:80]: you are right, I must've copied it from `ChangeSecurityChecks` will fix it

File: api/managementpb/checks.proto [hunk 3]
```diff
@@ -7,6 +7,14 @@ option go_package = "api/managementpb;managementpb";
 import "google/api/annotations.proto";
 import "managementpb/severity.proto";
 
+// SecurityCheckInterval represents possible execution interval values for checks.
+enum SecurityCheckInterval {
+  SECURITY_CHECK_INTERVAL_INVALID = 0;
+  STANDARD = 1;
+  FREQUENT = 2;
```
Comment (oter) [inline api/managementpb/checks.proto:14]: Can we add a short description for each interval, e.g. from where they come?

**Review by oter — APPROVED** [review]

File: api/managementpb/checks.proto (hunk 3 above)
Comment (palash25) [reply api/managementpb/checks.proto:14]: will add it

Comment (palash25) [reply api/managementpb/checks.proto:14]: on second thoughts all the intervals come from the same source (the UI in this case) so why do we need a comment for each of them. could you give me example of what you are looking for?

**Comment by palash25** [comment]
sorry for the force push, had to do it, since the conflicts were in a generated code, it got ugly

**Review by artemgavrilov — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Review by Dasio — APPROVED** [review]

**Review by atymchuk — APPROVED** [review]

**Pull request merged**